crates/biome_js_formatter/report.md

```diff
@@ -3225,8 +3444,22 @@
 
 
 ### js/import-assertions/empty.js
+```diff
+ export * as foo from "foo.json";
+-export * as bar from "bar.json" assert {};
+-export * as baz from "baz.json" /* comment */ assert {};
++export * as bar from "bar.json";
++export * as baz from "baz.json" /* comment */;
+ 
+ import * as foo from "foo.json";
+-import * as bar from "bar.json" assert {};
+-import * as baz from "baz.json" /* comment */ assert {};
++import * as bar from "bar.json";
++import * as baz from "baz.json" /* comment */;
```

Comment:
The fact that they remove empty assertions is... weird.

Reply:
Yeah. I asked about that here: https://github.com/prettier/prettier/pull/15757#issuecomment-2247732548